tests/e2e/hcp_external_auth_test.go

```diff
@@ -51,38 +51,61 @@ var _ = Describe("Rosacli Testing", func() {
 				_, err = rosaClient.BreakGlassCredential.RetrieveHelpForDelete()
 				Expect(err).ToNot(HaveOccurred())
 
-				By("Create a break-glass-credential to the cluster")
-				userName := common.GenerateRandomName("bgc-username", 2)
+				By("Create a break-glass-credential to the cluster with both --username and --expiration flag")
+				userName_1 := common.GenerateRandomName("bgc-username", 2)
 				createTime := time.Now().UTC()
 				expiredTime := createTime.Add(2 * time.Hour).Format("Jan _2 2006 15:04 MST")
 
-				resp, err := rosaClient.BreakGlassCredential.CreateBreakGlassCredential(clusterID, userName, "--expiration", "2h")
+				resp, err := rosaClient.BreakGlassCredential.CreateBreakGlassCredential(clusterID, "--username", userName_1, "--expiration", "2h")
 				Expect(err).ToNot(HaveOccurred())
 				textData := rosaClient.Parser.TextData.Input(resp).Parse().Tip()
 				Expect(textData).To(ContainSubstring("INFO: Successfully created a break glass credential for cluster '%s'", clusterID))
 
+				By("Create a break-glass-credential to the cluster with only --username flag")
+				userName_2 := common.GenerateRandomName("bgc-user", 2)
+
```

Comment:
I think it would be better to use a map for the different requests and check every credential parameters after creation.